### DISCUSSION TITLE
Styles not rendered SS after run production build

### DISCUSSION THREAD

**Discussion by Mlobaievskyi** [vercel/next.js#25722] [Help]
### What version of Next.js are you using?

10.2.0

### What version of Node.js are you using?

14.17.0

### What browser are you using?

Firefox, Safari, Chrome

### What operating system are you using?

macOS

### How are you deploying your application?

next start

### Describe the Bug

I have created next.js application using npx create-next-app. After that, I try to create and run a production build to check if everything is ok.

`npm run build`
`npm start`

Everything builds successfully but I notice that styles not rendered on the server, but work client side.

![Screenshot 2021-05-13 at 00 39 56](https://user-images.githubusercontent.com/19978874/118052846-c610a780-b383-11eb-8ea7-701ab2811c5d.png)

### Expected Behavior

SSR load styles same as client

![Screenshot 2021-05-13 at 00 41 15](https://user-images.githubusercontent.com/19978874/118052950-f0626500-b383-11eb-9a56-d6fca9b7c5b9.png)

### To Reproduce

`npx create-next-app`
`npm run build`
`npm start`

- Check if server render styles correctly

**Comment by AriaMoradpour** [comment]
have you tried `next build`? or what does your `build` script run?

**Comment by Mlobaievskyi** [comment]
> have you tried `next build`? or what does your `build` script run?

My build script does exactly the same
```
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start"
  },
```

**Comment by Mlobaievskyi** [comment]
Upgrade to next `10.2.3` didn't help. Also tried switch webpack 4 to 5 and still the same problem. It loads the HTML tree without styles pre-rendered.

**Comment by timneutkens** [comment]
A `<link>` tag is in the HTML which loads the styles. This is not a bug.

**Comment by simon360** [comment]
I've just tried this with a fresh `npx create-next-app`.

If JavaScript is disabled and I'm running `npm run dev`, then styles _don't_ load. But that's hardly surprising, tbh.

If JavaScript is disabled and I'm running `npm run start` (after an `npm run build`), though, styles seem to load fine (via a `<link />`, as @timneutkens suggested).
